notebook/notebookapp.py

```diff
@@ -1697,6 +1705,67 @@ def remove_server_info_file(self):
             if e.errno != errno.ENOENT:
                 raise
 
+    def write_browser_open_file(self):
+        """Write an nbserver-<pid>-open.html file
+
+        This can be used to open the notebook in a browser
+        """
+        # default_url contains base_url, but so does connection_url
+        open_url = self.default_url[len(self.base_url):]
+
+        with open(self.browser_open_file, 'w', encoding='utf-8') as f:
+            self._write_browser_open_file(open_url, f)
+
+    def _write_browser_open_file(self, url, fh):
+        if self.token:
+            url = url_concat(url, {'token': self.token})
```

Comment:
This isn't using the one-time token. That means the one-time token is never used. Should we be using it here, or get rid of the one-time token altogether?

Reply:
My thinking was to get rid of it, and have a file which can be used to authenticate multiple browsers to the same notebook. I came up with the one-time token to mitigate the issue with command line arguments being visible to other users. Now that we have a better solution to that, it should be unnecessary.

I'll add a commit in a minute to get rid of it.